repos/system_upgrade/common/models/custommodifications.py

```diff
@@ -0,0 +1,12 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class CustomModifications(Model):
+    """Model to store any custom or modified files that are discovered in leapp directories"""
+    topic = SystemFactsTopic
+
+    filename = fields.String()
+    actor_name = fields.String()
+    type = fields.StringEnum(choices=['custom', 'modified'])
+    rpm_checks_str = fields.String(default='')
```

Comment:
I have been thinking about it and decided to go with single message per file but cumulative (1 message for all changed / added  files in the Report). That approach is more extensible imho in case we need to track anything else.
So if it's not a hard blocker I'd rather leave it as is.

Reply:
Oki, no problem. It's possible that we will later change some names to be more specific, but cannot say now yet. I will wait in this case for the final approach to see how it looks like once the files from the framework and configuration files are covered too.

Reply:
I think the name could be a little more specific. On it's own it's not clear what CustomModifications are.

Reply:
Ohh naming :) I have been thinking about alternative names for what is now a `CustomModifications` model - what do you think about `FileCustomisations` or `FileModifications`?

Reply:
what about `LeappFileCustomisations`? or `CustomUpgradeModifications? anyway, I am ok to go now with CustomModifications now and change the model name in a followup if we want - until we create new upstream release, we can still rename it without starting the deprecation process.